src/main/java/org/jenkinsci/pipeline_steps_doc_generator/PipelineStepExtractor.java

```diff
@@ -158,8 +158,9 @@ public static void main(String[] args){
                                     Class<?> delegateOptionType = delegateOptionSchema.getType();
                                     Descriptor<?> delegateDescriptor = Jenkins.getInstance().getDescriptor(delegateOptionType.asSubclass(Describable.class));
                                     // TODO for some reason, stepsToPlugin contains entries for descriptors but not their describables:
```

Comment:
Update comment if stripping out stepsToPlugin, and explain what's going on here

Reply:
The comment is now removed and there are some comments for the new code.